R/format_PEW_temp.R

```diff
@@ -0,0 +1,419 @@
+#'   janitor::clean_names(case = "upper_camel") %>%
+#'   janitor::remove_empty(which = "rows") %>%
+#'   #### Solve NA values (some are explicitly stated as character "NA")
+#'   dplyr::mutate(across(where(is.character), .fns = ~replace(., . ==  "NA" , NA))) %>%
+#'   #### Convert to corresponding format
+#'   dplyr::mutate(Year = as.integer(Year),
+#'                 Date = as.Date(Date),
+#'                 Tarsus = as.numeric(Tarsus),
+#'                 Mass = as.numeric(Mass),
+#'                 TarsusPartner = as.numeric(TarsusPartner),
+#'                 MassPartner = as.numeric(MassPartner),
+#'                 ClutchSize = as.integer(ClutchSize),
+#'                 DateEgg1 = as.Date(DateEgg1),
+#'                 HatchDateD0 = as.Date(HatchDateD0),
+#'                 NumberOfRingedChicks = as.integer(NumberOfRingedChicks),
+#'                 DateRingingChicks = as.Date(DateRingingChicks),
+#'                 NoOfChicksD3 = as.integer(NoOfChicksD3),
+#'                 BroodMassD3 = as.numeric(BroodMassD3),
+#'                 ObservationTimeH = as.numeric(ObservationTimeH),
+#'                 Seks = substr(Seks, 1, 1),
+#'                 Species = "CYACAE",
+#'                 PopID = "PEW") %>%
+#'   #### Rename variables to standardized format
+#'   dplyr::rename(BreedingSeason = Year,
+#'                 IndvID = Id,
+#'                 Sex =  Seks,
+#'                 NestboxID = Nest,
+#'                 ObserverID = Measurer) %>%
+#'   #### Remove columns which we do not store in the standardized format
+#'   dplyr::select(-FeatherCollection ,
+#'                 -BreathRateTime50Breaths,
+#'                 -FeathersPartner,
+#'                 -BreathRatePartnerTime50Breaths,
+#'                 -BloodSample,
+#'                 -TimeBloodSample,
+#'                 -BloodSampleDuration,
+#'                 # -NewRing,
+#'                 -VisitRateVisitsH,
+#'                 -VrPartner,
+#'                 -VisitsAlternated,
+#'                 -VisitsAlternatedPartner,
+#'                 -VisitsSync10,
+#'                 -ChickAgeOfBehavObserv,
+#'                 -MateStrategy) %>%
+#'   #### Reorder columns
+#'   dplyr::select(BreedingSeason,
+#'                 Species,
+#'                 PopID,
+#'                 IndvID,
+#'                 Sex,
+#'                 Age,
+#'                 everything())
+#'
+#'
+#'
+#'
+#' #### check columns with NAs
+#' # sort(apply(pew_data, 2, function(x) sum(is.na(x))), decreasing = TRUE)
+#'
+#'
+#'
+#' #### --------------------------------------~
+#' #### INDIVIDUAL DATA                    #### done, need to check
+#' #### --------------------------------------~
+#'
+#' #### Still to solve:
+#' # There are some individuals (for example 13617043), where the rows of data related
+#' # to the same individual seem complementary, but are in two rows with
+#' # insufficient information when treating the rows separately.
+#' # How to deal with that?
```

Comment:
- Seems to me that there are some records that are not captures (i.e. they don't have a date) but just include brood information. So, I would explicitly remove the records with no date from Capture_data but still use them in Brood_data
- In other cases where there are multiple captures in a year (with dates) these should be treated as legitimate multiple captures.